### PR TITLE
Add gyroscope adapter

### PR DESCRIPTION
**Pull request by joshguha**

### CODE REVIEW

**Comment by slasher125** [comment]
@joshguha hi, ty for adding. seems like the adapter is not working yet. fee free to ping once its ready

**Comment by slasher125** [comment]
addind here https://github.com/DefiLlama/yield-server/commit/6231345790e15fdc8ee0d71885c415e9cc8dbd9e

File: src/adaptors/package-lock.json [hunk 1]
```diff
@@ -16081,6 +16249,24 @@
       "resolved": "https://registry.npmjs.org/encodeurl/-/encodeurl-1.0.2.tgz",
       "integrity": "sha512-TPJXq8JqFaVYm2CWmPvnP2Iyo4ZSM7/QKcSmuMLDObfpH5fi7RUGmd/rTDf+rut/saiDiQEeVTNgAmJEdAOx0w=="
     },
+    "encoding": {
+      "version": "0.1.13",
```
Comment (christianlantin) [inline src/adaptors/package-lock.json:16253]: +16253

File: src/adaptors/package-lock.json [hunk 2]
```diff
@@ -17294,6 +17480,31 @@
       "resolved": "https://registry.npmjs.org/isnumber/-/isnumber-1.0.0.tgz",
       "integrity": "sha512-JLiSz/zsZcGFXPrB4I/AGBvtStkt+8QmksyZBZnVXnnK9XdTEyz0tX8CRYljtwYDuIuZzih6DpHQdi+3Q6zHPw=="
```
Comment (christianlantin) [inline src/adaptors/package-lock.json:17481]: +17481

File: src/adaptors/package-lock.json [hunk 3]
```diff
@@ -18606,6 +18817,11 @@
       "resolved": "https://registry.npmjs.org/json-stringify-safe/-/json-stringify-safe-5.0.1.tgz",
```
Comment (christianlantin) [inline src/adaptors/package-lock.json:18817]: +18817

File: src/adaptors/package-lock.json [hunk 4]
```diff
@@ -21327,6 +21543,11 @@
         }
```
Comment (christianlantin) [inline src/adaptors/package-lock.json:21543]: +21543

**Pull request closed without merging**